src/scratch/data_navn/README.md

```diff
@@ -0,0 +1,227 @@
+---
+title: Lærerveiledning - Bruk data: Jente- og guttenavn
+language: nb
+---
+
+# Om oppgaven {.activity}
+I denne oppgaven skal elevene hente data fra SSB og lage en enkel spørrefunksjon for å sjekke om navn er jente- eller guttenavn.
+
+## Oppgaven passer til: {.check}
+ __Fag__: Matematikk, Norsk
+
+__Trinn__: 2.-7. trinn
+
+__Tema__: Statistikk, Databehandling
+
+__Nivå__: Nybegynner
+
+__Tidsbruk__: Dobbelttime eller mer.
+
+##
```

Comment:
Hvorfor har du flere av disse? `<section>` avsluttes når en ny overskrift med samme nivå kommer, altså `## Kompetansemål {.challenge}` på linje 22 vil avslutte seksjonen som tilhører `## Oppgaven passer til: {.check}`. Hvis ikke er det en bug i [markdown-it-header-sections](https://www.npmjs.com/package/markdown-it-header-sections).